indy_node/server/node.py

```diff
@@ -223,10 +240,10 @@ def acknowledge_upgrade(self):
 
         self.startedProcessingReq(*request.key, self.nodestack.name)
         self.send(request)
-        self.upgrader.notified_about_upgrade_result()
+        self.upgrader.notified_about_action_result()
```

Comment:
This notification is made in `Upgrader` only. Thus `_action_started` is now reset to `False` in `Upgrader` class only. And this is wrong.

Reply:
done